collect_app/src/androidTest/java/org/odk/collect/android/AllWidgetsFormTest.java

```diff
@@ -190,15 +217,22 @@ public void testActivityOpen()  {
 
     //region Widget tests.
 
+    // Screengrab.screenshot("image-name") captures screenshot of widgets.
```

Comment:
This comment feels buried. I would add a Javadoc header to the whole file. Something like
```
/**
 * Integration test that runs through a form with all question types.
 *
 * screengrab (https://docs.fastlane.tools/actions/screengrab/) is used to generate screenshots for
 * documentation and releases. Calls to Screengrab.screenshot("image-name") trigger screenshot
 * creation.
 */
```
Once the documentation PR is merged you can link it from here as well.